Update README.md (#13)
OwshenNetwork/genesis: README.md

    ```

> 6. Run the Owshen Wallet and navigate to http://127.0.0.1:9000 in your browser.

Reviewer: Note: running `./Owshen_v0.1.0_x86_64.AppImage info` doesn't need the browser. 
```suggestion
6. Run the following command to get your "Owshen Address".
```

    ```bash
   ./Owshen_v0.1.0_x86_64.AppImage info
    ```

> 7. Copy your Owshen Address and put it as a comment under our post! https://x.com/OwshenNetwork/status/1739258666199449979

Reviewer: ```suggestion
Optionally, you can run the Owshen Wallet `./Owshen_v0.1.0_x86_64.AppImage wallet` and navigate to http://127.0.0.1:9000 in your browser to get your Owshen Address.
7. Copy your Owshen Address and put it as a comment under our post! https://x.com/OwshenNetwork/status/1739258666199449979
```

> 7. Copy your Owshen Address and put it as a comment under our post! https://x.com/OwshenNetwork/status/1739258666199449979

Reviewer: ```suggestion
Optionally, you could run Owshen Wallet by `./Owshen_v0.1.0_x86_64.AppImage wallet`  and navigate to http://127.0.0.1:9000 in your browser to get your Owshen Address a.
7. Copy your Owshen Address and put it as a comment under our post! https://x.com/OwshenNetwork/status/1739258666199449979
```
